generators/generator-conversational-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "generator-conversational-core",
-  "version": "1.0.12",
+  "name": "@microsoft/generator-microsoft-bot-conversational-core",
+  "version": "1.0.0-preview1",
```

Comment:
You can make this one "preview.1" as well

Reply:
For my benefit, are we going with preview.1, or preview1? I don't mind either way, but I just changed them in the other components to preview1. I don't mind either way, but now would be the time to make the call and pull the trigger.